src/main/java/cat/tecnocampus/webclassexample/webcontroller/WebControllerGets.java

```diff
@@ -51,6 +51,7 @@ public String getAllNotes(Model model) {
         return "getNoteS";
     }
 
+    //This is added to illustrate the capability of limiting the access of a resource to its user owner only
```

Comment:
May be you could have said: ... to its owner. The owner must obviously be a registered user